test/VoteManager.js

```diff
@@ -652,6 +652,146 @@ describe('VoteManager', function () {
         assertBNEqual(balanceAfterAcc10, balanceBeforeAcc10.add(slashPenaltyAmount.div('2')),
           'the bounty hunter should receive half of the slashPenaltyAmount of account 4');
       });
+
+      it('if reported value is zero,should be able to commit', async function () {
```

Comment:
remove this testcase. there is no "reported value" input in commit

Reply:
I thought votes array is used in commitment , so i should add this as a testCase that staker can commit or not if the reported value is zero. If you say , I will remove it.

Reply:
what do you mean by reported value?

Reply:
https://github.com/razor-network/contracts/pull/262/files#diff-f683c8d5b95777f9e626d97eb2b74a82ffe9f58b93fe5f4a6a704f90885120baR667

This array .